Multisig/UI/Transactions/TransactionDetails/TransactionDetailsView.swift

```diff
@@ -9,26 +9,69 @@
 import SwiftUI
 
 struct TransactionDetailsView: View {
+    
     @FetchRequest(fetchRequest: Safe.fetchRequest().selected())
     var selectedSafe: FetchedResults<Safe>
 
     let transaction: TransactionViewModel
+
+    @State
+    private var showsLink: Bool = false
+    private let padding:CGFloat = 11
+
     var body: some View {
         List {
             TransactionHeaderView(transaction: transaction)
+
+            if data != nil {
+                VStack (alignment: .leading) {
+                    BoldText("Data")
+                    ExpandableButton(title: "\(data!.0) Bytes", value: data!.1)
+                }.padding(.vertical, 11)
+            }
+
             TransactionStatusTypeView(transaction: transaction)
             if !(transaction.confirmations?.isEmpty ?? true) {
```

Comment:
don't we have the "hasConfirmations" property on the transaction view model? I think, it should be used here.

Reply:
fixed in the other pr already